src/components/forms/groups/Create.js

```diff
@@ -3,11 +3,61 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { reduxForm, Field, stopSubmit } from 'redux-form';
 
-import Input from 'components/Input';
+import styles from './styles/index.scss';
 import Button from 'components/Button';
 
 const CREATE_GROUP_FORM = 'createGroup';
 
+const renderInputField = ({ input, label, placeholder, meta: { touched, error } }) => (
+  <div>
+    <div>
+      <label>{label}</label>
+    </div>
+    <input className="form-control" {...input} placeholder={placeholder} />
+    {touched && <div className={`text-danger ${styles.error}`}>{error}</div>}
+  </div>
```

Comment:
But you're reimplementing our 'Input' field locally with "renderInputField". What's the difference? Are you proposing do this in each form across the app?

If `components/Input` doesn't feed your needs I would addapt it instead of duplicate it all over the app.

I think an input should look and have always the same behaviour. This way styles, error logic are only in that component.

My idea was that input component is the entry point for all kind of form inputs: text, selector, checkbox,... Inside we can split it into smaller components. But label, info, errors are the same for all of it.